### PR TITLE
refactor: Separate 'git' and 'fs' util modules

### PR DESCRIPTION
**Pull request by zharinov**
Ref #6617

### CODE REVIEW

File: lib/util/fs/index.spec.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { getName } from '../../../../test/util';
-import { readLocalFile } from '.';
+import { getName } from '../../../test/util';
+import { readLocalFile } from './index';
```
Comment (viceice) [inline lib/util/fs/index.spec.ts:2]:
```suggestion
import { readLocalFile } from '.';
```

File: lib/util/git/index.spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import fs from 'fs-extra';
 import Git from 'simple-git/promise';
 import tmp from 'tmp-promise';
-import * as git from '.';
+import * as git from './index';
```
Comment (viceice) [inline lib/util/git/index.spec.ts:4]:
```suggestion
import * as git from '.';
```

**Review by viceice — CHANGES_REQUESTED** [review]
Just some small issues

File: lib/util/git/index.spec.ts (hunk 2 above)
Comment (rarkins) [reply lib/util/git/index.spec.ts:4]: I'll fix

**Pull request merged**